packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -569,7 +571,7 @@ describe('plugin-meetings', () => {
               await setup(mediaType, false, false);
               muteState.ignoreMuteStateChange = true;
 
-              muteState.handleLocalStreamMuteStateChange(meeting, true);
+              muteState.handleLocalStreamMuteStateChange(meeting, false);
```

Comment:
same here, this value should not have been reversed, it should remain as true. With value false, this test makes no sense, because we're not actually changing the mute value (it's initialized to false), so if we comment out the code from line 572, the test still passes, which defeats the point of this test that's supposed to check that when `ignoreMuteStateChange` is true then we don't call `MeetingUtil.remoteUpdateAudioVideo`

Reply:
Done.